src/sentry/static/sentry/app/components/smartSearchBar/index.tsx

```diff
@@ -837,34 +952,28 @@ class SmartSearchBar extends React.Component {
           },
         });
       });
-    } else {
-      pinSearch(
-        api,
-        organization.slug,
-        savedSearchType,
-        removeSpace(this.state.query)
-      ).then(resp => {
-        if (resp && resp.id) {
-          browserHistory.push({
-            ...router.location,
-            pathname: `/organizations/${organization.slug}/issues/searches/${resp.id}/`,
-            query: currentQuery,
-          });
-        }
-      });
+      return;
     }
 
-    trackAnalyticsEvent({
-      eventKey: 'search.pin',
-      eventName: 'Search: Pin',
-      organization_id: organization.id,
-      action: !!pinnedSearch ? 'unpin' : 'pin',
-      search_type: savedSearchType === 0 ? 'issues' : 'events',
-      query: pinnedSearch?.query ?? this.state.query,
+    const resp = await pinSearch(
+      api,
+      organization.slug,
+      savedSearchType,
+      removeSpace(this.state.query)
+    );
```

Comment:
Do you need to handle any network errors here?

Reply:
We weren't before, didn't want to add too much new stuff